jwst/jump/jump_step.py

```diff
@@ -70,7 +78,11 @@ def process(self, input):
             result = run_detect_jumps(input_model, gain_model, readnoise_model,
                                       rej_thresh, three_grp_rej_thresh, four_grp_rej_thresh, max_cores,
                                       max_jump_to_flag_neighbors, min_jump_to_flag_neighbors,
-                                      flag_4_neighbors)
+                                      flag_4_neighbors,
```

Comment:
Same  comment as above: I don't think it's necessary to equate all the new values to argument names, just pass them in the correct order. Of course it doesn't hurt to do it this way; just takes up more real estate in the code.

Reply:
Fixed.